docs/security/ssl/obtaining-lets-encrypt-certificates-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,190 @@
+---
+author:
+    name: 'Sean Webber'
+    email: 'swebber@yazzielabs.com'
+description: 'Installing Let\'s Encrypt and obtaining SSL certificates on Ubuntu 14.04 LTS'
+keywords: '14.04,ACME,free,HTTPS,Let\'s Encrypt,LTS,SSL,Ubuntu'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'N/A'
+modified: 'Friday, January 1st, 2016'
+title: 'Obtaining Let\'s Encrypt Certificates on Ubuntu 14.04'
+contributor:
+    name: 'Sean Webber'
+    link: 'https://github.com/seanthewebber'
+---
+
+## Introduction
+
+Let's Encrypt is an SSL certificate authority managed by the Internet Security Research Group (ISRG). It utilizes the Automated Certificate Management Environment (ACME) to automatically deploy browser-trusted SSL certificates to anyone for free.
+
+This tutorial will cover the following on Ubuntu 14.04:
+- Installing the Let's Encrypt ACME client
+- Obtaining Let's Encrypt certificates
+- Required attention and maintenance
+- Technical details about Let's Encrypt and certificates issued by Let's Encrypt
+
+{: .caution}
+>
+> As of 2016-01-01, Let's Encrypt is still in *public beta*. Although most users have reported success with Ubuntu 14.04, the ACME client is still being debugged and developed. **Do not deploy Let's Encrypt Public Beta in a production environment without testing it beforehand.**
+
+## Prerequisites
+
+- A Linode server running Ubuntu 14.04 with < 2GB of RAM. On a **Linode 1024** server, add 1GB of SWAP to meet this requirement. We recommend following our [Getting started](/docs/getting-started) guide if you need help setting up your Linode
```

Comment:
I think that the correct way is
`A Linode server running Ubuntu 14.04 with > (more than) 2GB of RAM.`
because the greater amount is the one you have in the VM, the one that should be higher than 2GB.

I would probably rephrase it as
`A Linode server running Ubuntu 14.04 with at least 2GB of RAM.`